src/millie.nim

```diff
@@ -10,46 +10,45 @@ const units = @[
   "P", # quadrillion
 ]
 
-## Converts big numbers to what's pleasant to see (an adorable, little girl, perhaps?)
+
 proc millify*(
-    value: int64,
+    value: SomeNumber,
```

Comment:
`SomeNumber` in Nim is a type class matching all possible number types.
This type makes the whole procedure a generic, because it'll generate different code based on the type of the number.